ui/lib/apps/KeyViz/index.meta.ts

```diff
@@ -1,9 +1,10 @@
 import { EyeOutlined } from '@ant-design/icons'
+import translations from './translations'
 
 export default {
   id: 'keyviz',
   routerPrefix: '/keyviz',
   icon: EyeOutlined,
-  translations: require.context('./translations/', false, /\.yaml$/),
+  translations,
   reactRoot: () => import(/* webpackChunkName: "app_keyviz" */ '.'),
```

Comment:
Maybe `webpackChunkName` is not respected and needed?

Reply:
Yep, I try to keep webpack build and esbuild build both work as possible as I can, but it seems meaningless.